src/utils/xmp.js

```diff
@@ -0,0 +1,66 @@
+import { LICENSES, licenseSlug } from '@/utils/license-utilities'
+
+function minify(data) {
+    return data.replace(/ {2,}/gi, '').replace(/\n/gi, '')
+}
+
+export const createXMP = ({ shortName, workUrl = '', workTitle = '', creatorName = '', lang = 'en-US' }) => {
+    const slug = licenseSlug(shortName).replace(/-/gi, '_').toUpperCase()
+
+    const copyrighted = shortName !== LICENSES.CC0.SHORT
+    const xapRights = copyrighted
+        ? `<rdf:Description rdf:about='' xmlns:xapRights='http://ns.adobe.com/xap/1.0/rights/'>
+             <xapRights:Marked>true</xapRights:Marked></rdf:Description>`
```

Comment:
I believe the [XMP spec](https://wwwimages2.adobe.com/content/dam/acom/en/devnet/xmp/pdfs/XMP%20SDK%20Release%20cc-2016-08/XMPSpecificationPart1.pdf#G5.869889) says booleans are either exactly `True` or `False` (first char uppercase).

Reply:
Correct, I haven't noticed that before. Thank you!